app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -714,6 +715,15 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
                 Wallpaper.nameIsDefault(settings.currentWallpaperName)
 
             defaultWallpaper.set(isDefaultTheCurrentWallpaper)
+
+            @Suppress("TooGenericExceptionCaught")
+            try {
+                notificationsAllowed.set(
+                    NotificationManagerCompat.from(applicationContext).areNotificationsEnabled(),
```

Comment:
This is good to have, but I've realized that we should also record if we the specific channel we're interested in is enabled (and possibly do that directly in the worker). There's a different API for that via the notification channel. Let's add that on Monday before landing. Happy to help with that.

Reply:
Sounds good.  I will look into doing that. Thanks